.github/workflows/release.yml

```diff
@@ -0,0 +1,33 @@
+---
+name: Release ansible-rulebook
+
+env:
+  LC_ALL: "C.UTF-8" # prevent ERROR: Ansible could not initialize the preferred locale: unsupported locale setting
+
+on:
+  workflow_dispatch:
+
+jobs:
+  stage:
+    runs-on: ubuntu-latest
```

Comment:
Can you add `if: github.repository == 'ansible/ansible-rulebook'` to make clear this workflow is not intended to be executed by forks?

Reply:
It doesn't work on forks because it doesn't have permissions to push to this repo.